tags/info/olderversions.tag

```diff
@@ -1,5 +1,8 @@
 type: text
+aliases: oldversion, oldversions, olderversion
 
 ---
 
-Geyser currently only officially supports servers running on 1.12.2+ or higher. If the server is below the latest version of the game then ViaVersion must also be installed. You can still join servers below 1.12.2 if you run Geyser on a proxy such as Velocity or as the Standalone platform, but keep in mind that you likely won't get support on these versions irregardless of how you're running it. Versions like 1.8 are over half a decade old, and we highly recommend updating to a later version of the game. Read [this](https://madelinemiller.dev/blog/which-minecraft-version/) post for more information regarding which version is best for you.
+Geyser currently only officially supports servers running on 1.12.2+ or higher. If the server is below the latest version of the game then ViaVersion must also be installed.
+
+You can still join servers below 1.12.2 if you run Geyser on a proxy such as Velocity or as the Standalone platform, but keep in mind that you likely won't get support on these versions irregardless of how you're running it. Versions like 1.8 are over half a decade old, and we highly recommend updating to a later version of the game. Read [this](https://madelinemiller.dev/blog/which-minecraft-version/) post for more information regarding which version is best for you.
```

Comment:
```suggestion
You can still join servers below 1.12.2 if you run Geyser on a proxy such as Velocity or the Standalone platform, but keep in mind that you likely won't get support on these versions regardless of how you're running it. Versions like 1.8 are over half a decade old, and we highly recommend updating to a later version of the game. Read [this](https://madelinemiller.dev/blog/which-minecraft-version/) post for more information regarding which version is best for you.
```
fixed typo, also removed the extra `as` (although that might be incorrect)

Reply:
Redned created this tag so I'll leave this for now

Reply:
![image](https://user-images.githubusercontent.com/13787163/121820544-598f0e00-cc8b-11eb-99bb-6df0f07ab51a.png)
It is a word but it's non-standard apparently